mPower2/MotorControl/ViewControllers/MCTOverviewStepViewController.swift

```diff
@@ -37,6 +37,9 @@ open class MCTOverviewStepViewController : RSDOverviewStepViewController {
     
     public static let firstRunKey = "isFirstRun"
     
+    /// The constraint that sets the scroll bar's top background view's height.
+    @IBOutlet weak var scrollViewBackgroundHeightConstraint: NSLayoutConstraint!
```

Comment:
nit: 99% of the time this will work. It will **not** work if the user gets a phone call b/c that pushes the status bar down.